### PR TITLE
python: more tolerant assignment parsing

### PR DESCRIPTION
**Pull request by tsiq-karold**

### CODE REVIEW

File: znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md [hunk 1]
```diff
@@ -1 +1 @@
-* Fix [variable extraction](snippets/python#content-full-content-variable) crashing in certain cases by adding more checks 
+* Fix [variable extraction](snippets/python#content-full-content-variable) crashing in certain cases by directly parsing the code rather than using the AST
```
Comment (MykolaGolubyev) [inline znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md:1]: I cut 53 release already, could you please move to 1.54? Sorry I forgot to update GitHub release

Comment (tsiq-karold) [reply znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md]: Ah, didn't realise.  Updated.

Comment (MykolaGolubyev) [reply znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md]: We need to move it to a separate `1.54` dir and do what you did to setup a new entry in release notes (I can do this part myself later).
I suggest we replace `fix` with the parsing of class variables - new feature

Comment (tsiq-karold) [reply znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md]: That's exactly what I've done.

Comment (MykolaGolubyev) [reply znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md]: Ah sorry! was looking at outdated commit :( thank you

**Review by MykolaGolubyev — APPROVED** [review]

**Pull request merged**